File: Sandbox.Modules/General/HelloModule.cs
using Sandbox.Core;

namespace Sandbox.Modules.CSharp.General {
    [SandboxModule("Hello World", "hello", "A simple module to test using reflection to discover modules.")]
    public class HelloModule : SandboxModule {
        protected override void Execute() {
            OnExecutionStarted();
            SendResponse(SandboxEventType.None, "Hello World!");
            OnExecutionCompleted();
        }
    }
}

File: Sandbox.Modules.FSharp/General/HelloModule.fs
namespace Sandbox.Modules.FSharp.General

open Sandbox.Core

[<SandboxModule("Hello World", "fs-hello", "This module is a simple test of integrating the F# module collection with the module service.")>]
[<ModuleTags("general", "sandbox", "testing")>]
type DemoModule() =
   inherit SandboxModule()
   override b.Execute() = b.SendResponse(SandboxEventType.None, "Hello from F#!")